ambry-router/src/main/java/com.github.ambry.router/NonBlockingRouterMetrics.java

```diff
@@ -83,6 +80,10 @@
   public final Histogram putManagerHandleResponseTimeMs;
   public final Histogram getManagerHandleResponseTimeMs;
   public final Histogram deleteManagerHandleResponseTimeMs;
+  // time spent in getting a chunk filled once it is available. This is reported per chunk.
+  public final Histogram chunkFillTimeMs;
```

Comment:
why is this reported per chunk but the other one across all chunks?

won't reporting across all chunks be a little bad since it will essentially be a function of number of chunks?